### ISSUE TITLE
Peer review - Lab10 (By Francesco Volpi s318931)

### ISSUE THREAD

**Issue by Francesco1102** [giuliuss/computational-intelligence-2023#8]
Hi, your work is very interesting. I appreciate the comments that make the code understandable, and the graph at the end is a nice addition. The validation phase is well-executed; my only suggestion is that increasing the number of games during both the training and evaluation phases might provide more reliable statistics and produce stronger agents. If it takes too long, consider take into account symmetries to reduce the number of actions in each state.
In summary, well-done work, and good luck with your exams!
